pom.xml

```diff
@@ -675,10 +676,6 @@
                             <phase>verify</phase>
                             <configuration>
                                 <bundledSignatures>
-                                    <!--
-                                      this will automatically choose the right
-                                      signatures based on 'maven.compiler.target'
-                                    -->
                                     <bundledSignature>jdk-deprecated</bundledSignature>
```

Comment:
I cannot remember why we would allow system.out or things from other bundledSignature files in tests...Was there a specific reason for that, or should we make the rules stricter here as well ?